src/app/[locale]/profile/page.tsx

```
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function ProfilePage() {
+  return (
+    <div>PROFILE</div>
+  )
+}
```

ESLint

[error] Missing semicolon.

(semi)